crates/bevy_animation/src/lib.rs

```diff
@@ -348,6 +599,34 @@ pub enum RepeatAnimation {
     Forever,
 }
 
+/// Why Bevy failed to evaluate an animation.
+#[derive(Clone, Debug)]
+pub enum AnimationEvaluationError {
```

Comment:
Not a blocker since these errors already have better documentation than the average bevy module, but it might be helpful if the errors cases contained data about what component or property wasn't present / the keyframe array size. I don't if these are obvious from context or not, but I imagine it would be annoying to just get a `ComponentNotPresent` error without knowing which component it is.